collections/Laud_Misc/MS_Laud_Misc_159.xml

```diff
@@ -38,31 +37,154 @@
+                        <rubric>Expositum in iesu naue</rubric>
+                        <incipit><locus>(fol. 127v)</locus> Post mortem ergo moysi loquitur dominus</incipit>
+                        <explicit>et in parius sibi resistentibus discit quod ex se maiora non subigit.</explicit>
+                        <finalRubric>Explicit liber iesu naue</finalRubric>
+                     </msItem>
+                     <msItem>
+                        <locus>(fols. 130v–134v)</locus>
+                        <title>In Iudices</title>
+                        <rubric>Incipiunt capitula libri iudicum</rubric>
+                        <finalRubric>Expliciunt capitula</finalRubric>
+                        <rubric>Expositum de libri iudicum</rubric>
+                        <incipit>Historia iudicum non parua musteriorum indicat sacramenta</incipit>
+                        <explicit>et reparato certamine in finem fortissime a demonibus triumphabit.</explicit>
+                     </msItem>
+                     <msItem>
+                        <locus>(fols. 134v–135r)</locus>
+                        <title>In Ruth</title>
+                        <rubric>Incipit explanatio libri ruth.</rubric>
+                        <incipit>Videamus nunc et ruth. habet enim ista typum ecclesiae</incipit>
+                        <explicit>quae adimpleuit is qui pollicitus est per filium suum iesum cristum regem et saluatorem nostrum. cum quo est illi honor et gloria in saecula saeculorum. Amen.</explicit>
+                        <finalRubric>Explicit expositio libri ruth feliciter</finalRubric>
+                     </msItem>
+                     <msItem>
+                        <locus>(fol. 135v)</locus>
+                        <note>Various biblical quotations written as pen trials.</note>
+                     </msItem>
                   </msItem>
                </msContents>
                <physDesc>
                   <objectDesc form="codex">
                      <supportDesc material="perg">
                         <support>parchment</support>
+                        <extent>i + 137 + i leaves <dimensions type="leaf" unit="mm">
+                              <height>365</height>
+                              <width>275</width>
+                           </dimensions>
+                        </extent>
+                        <foliation>Modern foliation.</foliation>
+                        <collation>Mostly in quaternions. First leaf (with title page) lost; one quire lost after after fol. 39. <signatures>Quires signed on the final page: <q>I</q> (fol. 7v), <q>Q II</q> (15v), <q>III</q> (fol. 23v), <q>Q IIII–V</q> (fol. 31v, fol. 39v), <q>VII–VIII</q> (fols. 47v, 55v, 63v).</signatures></collation>
                      </supportDesc>
+                     <layoutDesc>
+                        <layout columns="2" writtenLines="36">Ruled in ink. 2 columns of 36 lines, ruled space <dimensions type="ruled" unit="mm">
```

Comment:
is this right? (ink ruling). seems so unusual at this date I thought I should check!

Reply:
That was a slip – thank you!